src/czbenchmarks/tasks/single_cell/perturbation_expression_prediction.py

```diff
@@ -80,7 +80,9 @@ class PerturbationExpressionPredictionOutput(TaskOutput):
 
 
 class PerturbationExpressionPredictionTask(Task):
-    display_name = "perturbation expression prediction"
+    display_name = "Perturbation Expression Prediction"
+    description = "Evaluate the quality of gene perturbation predictions using multiple classification and correlation metrics."
```

Comment:
```suggestion
    description = "Evaluate the quality of perturbation-induced changes in gene expression values using multiple classification and correlation metrics."
```

Reply:
Minor change since we (eventually) want to include more than just genetic perturbations. The Tahoe dataset contains chemical perturbations! :)